### DISCUSSION TITLE
Download backup is not supported

### DISCUSSION THREAD

**Discussion by dbenyakar** [Koenkk/zigbee2mqtt#18044] [General]
Trying to backup my Z2M configuration for Sonoff Zigbee 3 dongle using the backup option in settings-tools.
When I try to download the backup I get 'Not Supported' message. Same thing
![Screenshot_20230617_093141_Home Assistant](https://github.com/Koenkk/zigbee2mqtt/assets/5302121/e73396e3-3c38-4ef4-b751-0bc67f33a6ea)
 when I press Download State.
Any ideas?

**Comment by Koenkk** [comment]
You are probably using the Dongle-E (which does not support backups)

**Reply by dbenyakar** [reply]
Im using dongle-p. But I tried to do it on the mobile app. Maybe its not possible to do it on my mobile and it needs to be done on the PC?
